controllers/ReservationSystem.php

```diff
@@ -328,14 +328,35 @@ public function cancelBooking($channelId, $bookingId)
         $booking->addChild('booking_id', $bookingId);
 
         // Optionally add a note explaining why the booking is cancelled
-        $booking->addChild('note', 'Booking created accidentally');
+        $booking->addChild('note', $note);
 
         // Call TourCMS API, cancelling the booking
         $result = $tourCMSOperator->cancel_booking($booking, $channelId);
 
         return $result->booking;
     }
 
+    public function showCustomer($customerId, $channelId)
+    {
+        $this->listChannels();
```

Comment:
does not have sense to call list channels. Controller should handle it and cache

Reply:
listChannels checks if we have the channels cached and if not it also loads them, that's why before creating a tourcms operator I call to function, since the channel api key is required

Reply:
I agree. But this has no sense, it should be done in another place

Reply:
fixed